bin/cml/repo.js

```diff
@@ -0,0 +1,8 @@
+exports.command = 'repo';
```

Comment:
delete this file? We're sticking with `ci` (not `repo prepare`).

Reply:
One thing is breaking the command structure for a single command until we find an appropriate noun+verb alternative, and a whole different thing is breaking the project structure because of that. 🤔 If this pull request is “all about user experience” I guess that keeping a consistent project structure won't affect that goal. ⚔️

Reply:
do you mean `repo prepare` will be (for now) a hidden subcommand (the existence of which is necessary from a technical implementation standpoint)?

Reply:
Yes, exactly that.

Reply:
Note that the `cml ci` command's logic **could** be placed separately, breaking the `bin/cml/noun/verb.js` and `bin/cml/noun.js` structure without affecting the functionality.

There is not insurmountable technical reason that forces us to preserve the same directory structure for every command, but, to my mind, it simplifies code navigation and reduces cognitive load during development.

Reply:
> Yes, exactly that.

ok fine by me :)